src/main/java/gtPlusPlus/xmod/gregtech/common/tileentities/machines/multi/production/turbines/GT_MTE_LargeTurbine_Plasma.java

```diff
@@ -129,8 +129,15 @@ public boolean checkRecipeGeneric(ItemStack[] aItemInputs, FluidStack[] aFluidIn
                                     Float.MIN_NORMAL,
                                     ((GT_MetaGenerated_Tool) aStack.getItem()).getToolStats(aStack).getSpeedMultiplier()
                                             * GT_MetaGenerated_Tool.getPrimaryMaterial(aStack).mToolSpeed
-                                            * 50))
-                            * getSpeedMultiplier();
+                                            * 50));
+
+                        //Calculate total EU/t (as shown on turbine tooltip (Fast mode doesn't affect))
+                    double aEUPerTurbine = aTotalOptimalFlow
+                                    * 40 * 0.0105
+                                    * GT_MetaGenerated_Tool.getPrimaryMaterial(aStack).mPlasmaMultiplier
+                                    * (50.0f + (10.0f * ((GT_MetaGenerated_Tool) aStack.getItem()).getToolCombatDamage(aStack)));
```

Comment:
This line is ugly and should be extracted to a variable for better readability imo